configs/yolof/iter_yolof_r50_c5_8x8_1x_coco.py

```diff
@@ -0,0 +1,124 @@
+_base_ = [
```

Comment:
1. rename to yolof_r50_c5_8x8_iter-1x_coco.py
2. inherits from  yolof_r50_c5_8x8_1x_coco.py